blobfuse/utilities.cpp

```diff
@@ -358,21 +327,26 @@ int azs_getattr(const char *path, struct stat *stbuf)
 
     int res;
     int acc = access(mntPathString.c_str(), F_OK);
-    if (AZS_PRINT)
-    {
-        fprintf(stdout, "accessing mntPath = %s returned %d\n", mntPathString.c_str(), acc);
-    }
     if (acc != -1 )
     {
+        syslog(LOG_DEBUG, "Accessing mntPath = %s for get_attr succeeded; object is in the local cache.\n", mntPathString.c_str());
         //(void) fi;
         res = lstat(mntPathString.c_str(), stbuf);
-        if (AZS_PRINT)
+        if (res == -1)
         {
-            printf("LSTAT res = %d, errno = %d, ENOENT = %d\n", res, errno, ENOENT);
+            int lstaterrno = errno;
+            syslog(LOG_ERR, "lstat on file %s in local cache during get_attr failed with errno = %d.\n", mntPathString.c_str(), lstaterrno);
+            return -lstaterrno;
+        }
+        else
+        {
+            syslog(LOG_DEBUG, "lstat on file %s in local cache succeeded.\n", mntPathString.c_str());
```

Comment:
I wonder if we also had the function name prepended to these text. If I see this in my syslog, how do I know that this happened during a getattr call ?